### PR TITLE
[DOC-2898] Update delegate faqs

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update delegate faqs

For reviewers, a preview is available.

[Harness Delegate FAQs](https://65007b290e827d0071497e86--harness-developer.netlify.app/docs/faqs/harness-delegate-faqs)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/faqs/harness-delegate-faqs.md [hunk 1]
```diff
@@ -27,17 +27,17 @@ Harness Platform has two major components:
...
 The delegate updates automatically. The delegate installation also installs a **Watcher** program that checks the Harness cloud periodically for new versions.
 
 Watcher ensures there is exactly one delegate process of each published version running.
 
 If there is a published version that is not running, Watcher downloads the JAR file for that version securely over HTTPS, installs it, and updates the delegate connection to Harness Manager. There is no downtime.
 
-##### Self-Managed
+##### Delegate manual updates
 
 You update the delegate. When you install a delegate by downloading the delegate YAML file from Harness, you select the self-managed option.
```
Comment (pratmit) [inline docs/faqs/harness-delegate-faqs.md:43]: You select the manual update option

**Review by pratmit — COMMENTED** [review]
Added a comment

File: docs/faqs/harness-delegate-faqs.md [hunk 2]
```diff
@@ -48,31 +48,31 @@ Delegate images are on [Docker Hub](https://hub.docker.com/r/harness/delegate-im
 The delegate image is signed. You can check it by running this command (with the current version number):
 
 ```
-docker trust inspect --pretty harness/delegate:22.03.74407
+docker trust inspect --pretty harness/delegate:yy.mm.xxxxx
```
Comment (douglas-j-bothwell) [inline docs/faqs/harness-delegate-faqs.md:51]:
```suggestion
docker trust inspect --pretty harness/delegate:yy.mm.verno
```
Just a thought....

**Review by douglas-j-bothwell — DISMISSED** [review]
One minor sug, otherwise looks good!

**Pull request merged**